quil-rs/src/program/mod.rs

```diff
@@ -827,24 +827,18 @@ DECLARE ec BIT
 
     #[test]
     fn frame_blocking() {
-        let input = r#"DEFFRAME 0 "a":
-    HARDWARE-OBJECT: "hardware"
-
-DEFFRAME 0 "b":
-    HARDWARE-OBJECT: "hardware"
-
-DEFFRAME 1 "c":
-    HARDWARE-OBJECT: "hardware"
+        let input = "DEFFRAME 0 \"a\":
+\tHARDWARE-OBJECT: \"hardware\"
```

Comment:
This looks like hard tabs were injected? Is this intentional?

Reply:
I believe so, the commit this is reverting originally changed them from tabs to spaces: https://github.com/rigetti/quil-rs/commit/18081daa34acd090b1daaabf3c5a8c46d13f6993#diff-02ccf5616a656b995be52b5d351335deded8c166cf246b234a0bfa1f33162c1eL723